get-poetry.py

```diff
@@ -561,17 +569,21 @@ def make_bin(self):
             os.mkdir(POETRY_BIN, 0o755)
 
         if WINDOWS:
+            # implicit system encoding for poetry.bat
```

Comment:
These comments (and those in the other files) are not really necessary and add too much noise to the codebase

Reply:
I will remove those comments.

Reply:
There are still extraneous comments that are not required